sdk/ai/ai-inference-rest/samples/v1-beta/javascript/telemetry.js

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * Demonstrates how to get instrumentation by open telemetry.
+ *
+ * @summary get instrumentation by open telemetry.
+ */
+
+const { diag, DiagConsoleLogger, DiagLogLevel } = require("@opentelemetry/api");
+const { registerInstrumentations } = require("@opentelemetry/instrumentation");
+const { createAzureSdkInstrumentation } = require("@azure/opentelemetry-instrumentation-azure-sdk");
+const { ConsoleSpanExporter, NodeTracerProvider, SimpleSpanProcessor } = require("@opentelemetry/sdk-trace-node");
+
+diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.VERBOSE);
```

Comment:
is this intentional? probably not needed in the sample

Reply:
Removed